consensus/src/block_storage/pending_votes.rs

```diff
@@ -1,12 +1,15 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+//! PendingVotes store pending votes observed for a fixed epoch and round.
```

Comment:
use `///` instead

Reply:
why? That's a module-level doc : o

Reply:
hmm, isn't this documenting the module?

Reply:
oh i see, https://learning-rust.github.io/docs/a5.comments_and_documenting_the_code.html TIL